README.md

```diff
@@ -199,6 +199,61 @@ func HandleRequest(ctx context.Context, name string) (string, error) {
 }
 ```
 
+To use the xray calls directly with lambda - for use with sql for example - you must begin a new facade segment and use the resulting context in relevant context-enabled calls.
+
+```go
+package yourpkg
+
+import (
+    "context"
+    "fmt"
+    "github.com/aws/aws-xray-sdk-go/header"
+    "github.com/aws/aws-xray-sdk-go/xray"
+    "time"
+)
+
+func HandleRequest(ctx context.Context, name string) (string, error) {
+
+    facadeSegmentName := "facade"
+    traceIDtr, isTraced := ctx.Value("x-amzn-trace-id").(string)
+
+    msg := "Request not traced"
+
+    if isTraced {
+        msg = fmt.Sprintf("request traced with %s", traceIDtr)
+
+        header := header.FromString(traceIDtr)
+        newCtx, seg := xray.BeginFacadeSegment(ctx, facadeSegmentName, header)
```

Comment:
@underscorenygren You shouldn't need to call the BeginFacadeSegment as this is handled automatically by the go SDK.  The xray.SQL that you added, will ensure that the subsegments the db/sql calls are traced. Can you resubmit with the changes?